crates/oxc_ast/src/span.rs

```diff
@@ -2,6 +2,33 @@ use oxc_span::{GetSpan, Span};
 
 use crate::ast::*;
 
+macro_rules! impl_get_span {
```

Comment:
Let's have as less macro as possible in these crates, a simple copy paste is totally fine here.

I have a rule https://oxc.rs/docs/contribute/rules.html#development-policy

> Avoid heavy macros or generics or any Rust techniques that would penalize compilation speed.

It gets heavy when we repeat these.

Reply:
Removed